matter_server/common/helpers/api.py

```diff
@@ -1,5 +1,5 @@
 """Several helpers for the websockets API."""
-
+from __future__ import annotations
```

Comment:
look out for side effects here.. Does the utility still work with this import on python 3.9 ?
we are using the inspect module to grab annotations for functions, the big difference between py3.9 and py3.10 is if the output is strings or actual classes/types. I remember the future import messing with that..

Reply:
After we merge https://github.com/home-assistant-libs/python-matter-server/pull/147 I think we can go ahead with more confidence here if tests pass.

Reply:
I'm currently testing this branch to look for any side effects

Reply:
everything still works perfectly fine, tested with py3.10 and 3.9 and everything still working smooth, couldn't detect any side efefcts